visualizer.py

```diff
@@ -0,0 +1,97 @@
+import datetime as dt
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+from benchmark import PriceScheduleDataFrame
+
+
+def plot_power_schedule_capacity_and_prices(
+        price_schedule_df: PriceScheduleDataFrame,
```

Comment:
Would it be possible to add a different price schedule of a different market, such as the intraday or imbalance market? And how could I implement these? I hope that the revenues from the imbalance market allow me to earn my battery back quicker.

Reply:
The imbalance market can be encoded into the `PriceScheduleDataFrame` structure. The combination of these different markets is not yet in the scope of this project.